src/app/mail/mail-settings/mail-settings.component.ts

```diff
@@ -66,7 +66,7 @@ export class MailSettingsComponent implements OnInit, AfterViewInit, OnDestroy {
 
   timeZoneFilter = new FormControl('', []);
   timeZoneFilteredOptions: Observable<Timezone[] | void>;
-
+  isRecoveryEmailEdit: boolean;
```

Comment:
rename `isRecoveryEmailEdit ` to `isEditingRecoveryEmail`

Reply:
fixed typos